### PR TITLE
Minimize trust

### PR DESCRIPTION
**Pull request by t-nelson**
#### Problem

Too much trust

#### Summary of Changes

Minimize trust throughout

### CODE REVIEW

File: validator/src/main.rs [hunk 1]
```diff
@@ -1574,7 +1576,8 @@ pub fn main() {
         )
         .arg(
             Arg::with_name("no_untrusted_rpc")
-                .long("no-untrusted-rpc")
+                .alias("no-untrusted-rpc")
+                .long("no-nonarchetype-rpc")
```
Comment (mvines) [inline validator/src/main.rs:1580]: This double negation is rough. How about "--archetype-rpc-only" instead?

File: validator/src/main.rs [hunk 2]
```diff
@@ -1574,7 +1576,8 @@ pub fn main() {
...
                 .takes_value(false)
                 .help("Use the RPC service of trusted validators only")
```
Comment (mvines) [inline validator/src/main.rs:1582]: "trusted" 😱

File: validator/src/main.rs [hunk 3]
```diff
@@ -1744,7 +1747,8 @@ pub fn main() {
         )
         .arg(
             Arg::with_name("halt_on_trusted_validators_accounts_hash_mismatch")
-                .long("halt-on-trusted-validators-accounts-hash-mismatch")
+                .alias("halt-on-trusted-validators-accounts-hash-mismatch")
+                .long("halt-on-archetype-validators-accounts-hash-mismatch")
                 .requires("trusted_validators")
                 .takes_value(false)
                 .help("Abort the validator if a bank hash mismatch is detected within trusted validator set"),
```
Comment (mvines) [inline validator/src/main.rs:1754]: "trusted"

**Review by mvines — COMMENTED** [review]
Direction looks great.  I'm not sold on the use of the word "archetype" yet though, it feels a little too obscure even for native speakers, and I bet it'll trip up non-native English speakers.

Some other ideas:
* --friendly-validator
* --known-validator
* --established-validator 
* --benevolent-validator

We might need to roll into the #terminology channel to hash this out!

File: validator/src/main.rs (hunk 1 above)
Comment (CriesofCarrots) [reply validator/src/main.rs:1580]: I think `known` works well here.

**Comment by t-nelson** [comment]
> Direction looks great. I'm not sold on the use of the word "archetype" yet though, it feels a little too obscure even for native speakers, and I bet it'll trip up non-native English speakers.
> 
> Some other ideas:
> 
>     * --friendly-validator
> 
>     * --known-validator
> 
>     * --established-validator
> 
>     * --benevolent-validator
> 
> 
> We might need to roll into the #terminology channel to hash this out!

Hmm... None of these really communicate my spite for the underlying motivation of these changes :thinking:

File: validator/src/main.rs (hunk 2 above)
Comment (t-nelson) [reply validator/src/main.rs:1582]: I guess it's obvious that I made most of these changes with `sed` now :sweat_smile:

**Comment by t-nelson** [comment]
I went with `known` throughout

File: validator/src/main.rs (hunk 1 above)
Comment (t-nelson) [reply validator/src/main.rs]: It's now `--only-known-rpc`

**Review by mvines — DISMISSED** [review]

File: docs/src/clusters.md [hunk 4]
```diff
@@ -44,12 +44,19 @@ solana config set --url https://api.devnet.solana.com
 $ solana-validator \
     --identity validator-keypair.json \
     --vote-account vote-account-keypair.json \
-    --known-validator dv1LfzJvDF7S1fBKpFgKoKXK5yoSosmkAdfbxBo1GqJ \
+    --known-validator dv1ZAGvdsz5hHLwWXsVnM94hWf1pjbKVau1QVkaMJ92 \
```
Comment (lijunwangs) [inline docs/src/clusters.md:47]: --trusted-validator right? I use --know-validator in the master branch it does not like it

Comment (lijunwangs) [reply docs/src/clusters.md:47]: Sorry -- I did not get the full picture. You are introducing a new parameters..

Comment (t-nelson) [reply docs/src/clusters.md:47]: Ah!  Yeah sorry I should've pointed that out earlier.  The t-word is a swear in this industry :confused:

**Pull request merged**